### PR TITLE
Adds a "You are here" marker for station maps.

### PR DESCRIPTION
**Pull request by Iamgoofball**
This should help with locating yourself in a new or complex map.

### CODE REVIEW

File: code/game/objects/structures/signs/signs_maps.dm [hunk 1]
```diff
@@ -4,6 +4,12 @@
 	name = "station map"
 	desc = "A framed picture of the station."
 	max_integrity = 500
+	
+/obj/structure/sign/map/examine(mob/user)
+	..()
+	if(prob(1) || (SSevents.holidays && SSevents.holidays[APRIL_FOOLS])) // TODO: Enable at all times in hypercube-station
+		say("You are here.")
```
Comment (Jalleo) [inline code/game/objects/structures/signs/signs_maps.dm]: to_chat(user,

**Comment by Armhulen** [comment]
this should definitely help out new maps getting off the gr- oh god dammit goof

**Comment by Armhulen** [comment]
that being said, this is a great change

File: code/game/objects/structures/signs/signs_maps.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/objects/structures/signs/signs_maps.dm]: no, I want the map to say it

Comment (PKPenguin321) [reply code/game/objects/structures/signs/signs_maps.dm]: It has a speaker in it

Comment (Iamgoofball) [reply code/game/objects/structures/signs/signs_maps.dm]: Yes

**Review by ShizCalev — APPROVED** [review]

**Review by Cyberboss — CHANGES_REQUESTED** [review]
If it is audibly speaking that means it has power, so repath it to machinery and have it check is_operational() before speaking

**Comment by Bawhoppen** [comment]
I was like "Goof made a serious PR to help people?" and then I saw the files changed.

**Comment by ninjanomnom** [comment]
@Cyberboss that's a bit excessive for what's just a minor joke

File: code/game/objects/structures/signs/signs_maps.dm [hunk 2]
```diff
@@ -4,6 +4,12 @@
...
+		visible_message("points at [user].")
```
Comment (Cyberboss) [inline code/game/objects/structures/signs/signs_maps.dm]: Okay I'll relent on the other thing, but use the proper pointing procs

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/signs/signs_maps.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/game/objects/structures/signs/signs_maps.dm]: We have those? Cool, will do

**Comment by vuonojenmustaturska** [comment]
@tgstation/commit-access Don't string a guy hoping to get his bad joke merged along like this. 

@Iamgoofball let a maintainer know if you're going to make this into something actually useful.

**Comment by Iamgoofball** [comment]
just gotta find where we generalized pointing to so I can get the arrow and shit

File: code/game/objects/structures/signs/signs_maps.dm (hunk 2 above)
Comment (ninjanomnom) [reply code/game/objects/structures/signs/signs_maps.dm]: That's a /living proc you meme @Cyberboss

**Comment by ShizCalev** [comment]
@Iamgoofball https://github.com/tgstation/tgstation/blob/ecd0d8b53876eef464788969c7bd85199640b686/code/modules/mob/mob.dm#L307

**Comment by Iamgoofball** [comment]
i am not attached to this joke anymore

**Comment by adamkad1** [comment]
goof plz

**Pull request closed without merging**